src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.INVALID_HANDLE_VALUE.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        internal static readonly IntPtr INVALID_HANDLE_VALUE = new IntPtr(-1);
```

Comment:
It would be better to make this a property. (Generates better code.)

Reply:
Per your other feedback, I've just deleted this altogether.